### PR TITLE
scale to zero at 19 and scale back at 9

### PR DESCRIPTION
**Pull request by TheoPascoli**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by schwitzguebel — APPROVED** [review]

File: k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        start: "0 7 * * 1-5"
+        end: "0 17 * * 1-5"
+        desiredReplicas: "1"
+
+---
+
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: cse-import-ec-d2cc-export-scale
```
Comment (kahyami) [inline k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml:24]: replace export by exporter

File: k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  namespace: gridcapa-t
+spec:
+  scaleTargetRef:
+    name: cse-import-ec-d2cc-export
```
Comment (kahyami) [inline k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml:28]: the same here

**Review by kahyami — CHANGES_REQUESTED** [review]

File: k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml (hunk 2 above)
Comment (danielthirion) [reply k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml:74]: As discussed, the scaleTargetRef points to the existing deployment which is currently named **cse-import-ec-d2cc-export** all over gridcapa-deployment. Perhaps we could rename this everywhere in another ticket

File: k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml (hunk 1 above)
Comment (danielthirion) [reply k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml]: renaming done here

**Review by WalAmeni — APPROVED** [review]

**Review by kahyami — APPROVED** [review]

**Pull request merged**